src/js/player.js

```diff
@@ -2218,27 +2219,23 @@ class Player extends Component {
    *         The current video source when getting
    */
   src(source) {
-    if (source === undefined) {
```

Comment:
Ah, I see. I think it's better to keep it. Why do extra work if we know for sure we can exit early here?